oasislmf/cli/exposure.py

```diff
@@ -184,15 +185,19 @@ def action(self, args):
         elif output_level == 'item':
             summary_cols = ['output_id', portfolio_num, acc_num, loc_num, policy_num, 'coverage_type_id']
 
+        guls_df = guls_df.loc[:, summary_cols + ['loss_gul']]
+        guls_df.drop_duplicates(inplace=True)
+
         if not il and not ril:
-            all_losses_df = all_losses_df.loc[:, summary_cols + ['loss_gul']]
+            all_losses_df = guls_df.loc[:, summary_cols + ['loss_gul']]
+            all_losses_df.drop_duplicates(keep=False,inplace=True)
```

Comment:
A single space between the comma and `inplace=True` should be inserted, i.e.

    all_losses_df.drop_duplicates(keep=False, inplace=True)

(See PEP8 rule E231 - https://pep8.readthedocs.io/en/1.4.6/intro.html)